forge-gui/res/cardsfolder/upcoming/ravenous_gigamole.txt

```diff
@@ -0,0 +1,12 @@
+Name:Ravenous Gigamole
+ManaCost:3 B
+Types:Creature Mole Horror
+PT:2/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigMill | TriggerDescription$ When CARDNAME enters the battlefield, mill three cards. You may put a creature card from among the cards milled this way into your hand. If you don't, put a +1/+1 counter on CARDNAME. (To mill a card, player put the top card of your library into your graveyard.)
+SVar:TrigMill:DB$ Mill | NumCards$ 3 | Defined$ You | RememberMilled$ True | SubAbility$ DBChangeZone
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Hand | ChangeType$ Creature.YouOwn+IsRemembered | RememberChanged$ True | ForgetOtherRemembered$ True | Hidden$ True | Optional$ True | SubAbility$ DBCounter
```

Comment:
> If a replacement effect causes the milled cards to be exiled rather than go to the graveyard, this effect can still apply to the cards "milled this way" in exile.